views/notifications.pug

```diff
@@ -0,0 +1,34 @@
+extends layout
+
+block content
+  h3.title.spaced STAR WARS
+
+  p.spaced
+    | &nbsp;Receive update, you must!
+
+  .row
+    .col-md-12
+      form.form-horizontal(action='/notifications', method='POST')
+        .form-group
+          label.col-md-2.control-label Message
+          .col-md-10
+            textarea.form-control(name='message')
+            span.text-primary#validation_message
+
+        .form-group
+          label.col-md-2.control-label Movie
+          .col-md-10
+            select.form-control(name='movie')
+              option(value='episode_viii') Episode VIII
+              option(value='rogue_one') Rogue One
+              option(value='han_solo_spinoff') Han Solo Spinoff
+
+        .form-group
+          .col-md-10.col-md-offset-2
+            a.btn.btn-danger#reset(href='')
+              i.fa.fa-close
+              | &nbsp;Reset
```

Comment:
Yes we are. lol

I'll add a CSS class to handle this one, it couldn't be uglier, right?